src/sitesToTest.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright (c), CKSource Holding sp. z o.o. All rights reserved.
+ */
+const pingSiteData: Record<string, string[]> = {
+	websites: [
+		'https://ckeditor.com/',
+		'https://cksource.com/',
+		'https://onlinehtmleditor.dev/',
+		'https://onlinemarkdowneditor.dev/'
+	],
+	ckeditorOrigins: [
+		'https://ckeditor.com/docs/',
+		'https://ckeditor.com/ckeditor-5/builder/',
+		'https://ckeditor.com/cke4/addons/plugins/all',
+		'https://ckeditor.com/docs/ckfinder/ckfinder3/',
+		'https://origin.ckeditor.com/old/forums',
+		'https://ckeditor.com/ckfinder/demo/',
+		'https://ckeditor.com/partners/',
+		'https://ckeditor.com/technology-partners'
+	],
+	CDNs: [
+		'https://cdn.ckeditor.com/',
+		'https://download.cksource.com/',
+		'https://cdn.ckbox.io/ckbox/latest/ckbox.js',
+		'https://download.ckbox.io/ckbox/ckbox_2.4.0.zip'
+	],
+	internalCDNs: [
+		'https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/index.html',
+		'https://builder.ckeditor-dev.com/ckeditor-5/builder/',
+		'https://builder-nightly.ckeditor-dev.com/ckeditor-5/builder/'
```

Comment:
```suggestion
		'https://builder-nightly.ckeditor-dev.com/ckeditor-5/builder/'
		'https://builder-demo.ckeditor-dev.com/ckeditor-5/builder/'
```

A new url was added in the meantime 😅

Reply:
Added :)